funnel/views/proposal.py

```diff
@@ -184,9 +185,10 @@ def proposal_edit(profile, space, proposal):
 def proposal_status(profile, space, proposal):
     form = ProposalStatusForm()
     if form.validate_on_submit():
-        proposal.status = form.status.data
+        proposal._status = form.status.data
```

Comment:
Put a `TODO` marker here. This view needs to go. State changes should require transitions.